### ISSUE TITLE
AttributeError: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by ydroneaud** [FrostyX/tracer#143]
On Fedora 31, doing dnf upgrade today:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 66, in transaction
    tracer = TracerFacade(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 79, in __init__
    self.apps = self.get_apps(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 84, in get_apps
    return query.from_packages(packages).now().affected_applications().get()
  File "/usr/lib/python3.7/site-packages/tracer/query.py", line 72, in get
    return self._method(**self._kwargs)
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 126, in trace_affected
    if not self._applications.find('kernel').ignore and self._has_updated_kernel():
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 134, in _has_updated_kernel
    running = System.running_kernel_package()
  File "/usr/lib/python3.7/site-packages/tracer/resources/system.py", line 119, in running_kernel_package
    return System.package_manager().find_package(System.kernel_package_name(), os.uname()[2])
  File "/usr/lib/python3.7/site-packages/tracer/resources/PackageManager.py", line 67, in find_package
    return self.package_managers[0].find_package(pkg_name, search)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 119, in find_package
    self._load_package_info_from_hdr(package, hdr)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 201, in _load_package_info_from_hdr
    package.description = hdr[rpm.RPMTAG_SUMMARY].decode()
AttributeError: 'str' object has no attribute 'decode'

**Comment by FrostyX** [comment]
Hello @ydroneaud, thank you for the feedback. 
Are you able to 100% reproduce this issue or it happens just occasionally?

**Comment by progkix** [comment]
It is happening to me as well. i recently did a dnf upgrade and a dnf install and it happened on both commands.

I seem to remember seeing an update for dnf come thru before this began.

**Comment by ydroneaud** [comment]
@FrostyX it happens again tody
I'm also using updates-testing.

**Comment by FrostyX** [comment]
Aaah, the reproducer is `sudo tracer -s qtile` or whatever running application.
I will look at it today.

**Comment by disk0x** [comment]
For what's that worth, the bug is still happening at a fully updated Fedora 31 installation, reproducible always (any dnf cli transaction that calls tracer)

```
$ sudo dnf reinstall tracer
Last metadata expiration check: 1:03:58 ago on Wed 10 Jun 2020 08:36:19 CEST.
Dependencies resolved.
================================================================================
 Package               Architecture  Version               Repository      Size
================================================================================
Reinstalling:
 python3-tracer        noarch        0.7.3-2.fc31          updates        105 k

Transaction Summary
================================================================================

Total download size: 105 k
Installed size: 286 k
Is this ok [y/N]: y
Downloading Packages:
python3-tracer-0.7.3-2.fc31.noarch.rpm          4.3 kB/s | 105 kB     00:24    
--------------------------------------------------------------------------------
Total                                           4.1 kB/s | 105 kB     00:25     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Reinstalling     : python3-tracer-0.7.3-2.fc31.noarch                     1/2 
  Cleanup          : python3-tracer-0.7.3-2.fc31.noarch                     2/2 
  Running scriptlet: python3-tracer-0.7.3-2.fc31.noarch                     2/2 
  Verifying        : python3-tracer-0.7.3-2.fc31.noarch                     1/2 
  Verifying        : python3-tracer-0.7.3-2.fc31.noarch                     2/2 
Tracer:
 Call to Tracer API ended unexpectedly:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 66, in transaction
    tracer = TracerFacade(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 79, in __init__
    self.apps = self.get_apps(packages)
  File "/usr/lib/python3.7/site-packages/dnf-plugins/tracer.py", line 84, in get_apps
    return query.from_packages(packages).now().affected_applications().get()
  File "/usr/lib/python3.7/site-packages/tracer/query.py", line 72, in get
    return self._method(**self._kwargs)
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 126, in trace_affected
    if not self._applications.find('kernel').ignore and self._has_updated_kernel():
  File "/usr/lib/python3.7/site-packages/tracer/resources/tracer.py", line 134, in _has_updated_kernel
    running = System.running_kernel_package()
  File "/usr/lib/python3.7/site-packages/tracer/resources/system.py", line 119, in running_kernel_package
    return System.package_manager().find_package(System.kernel_package_name(), os.uname()[2])
  File "/usr/lib/python3.7/site-packages/tracer/resources/PackageManager.py", line 67, in find_package
    return self.package_managers[0].find_package(pkg_name, search)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 119, in find_package
    self._load_package_info_from_hdr(package, hdr)
  File "/usr/lib/python3.7/site-packages/tracer/packageManagers/rpm.py", line 201, in _load_package_info_from_hdr
    package.description = hdr[rpm.RPMTAG_SUMMARY].decode()
AttributeError: 'str' object has no attribute 'decode'

Please visit https://github.com/FrostyX/tracer/issues and submit the issue. Thank you
We apologize for any inconvenience

Reinstalled:
  python3-tracer-0.7.3-2.fc31.noarch                                            

Complete!

```

**Comment by FrostyX** [comment]
The fix was released to tracer-0.7.4 which is currently in testing.
https://bodhi.fedoraproject.org/updates/FEDORA-2020-2e33457aa0
